File: 1929. Concatenation of Array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] result = new int[2*nums.length];

        for(int i=0; i<result.length; i++){
            if(i<nums.length) result[i]=nums[i];
            else result[i]=nums[i-nums.length];
        }
        return result;
    }
}

File: 1929. Concatenation of Array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        results = [0] * 2 * n

        for i in range(n):
            results[i] = nums[i]
            results[i+n] = nums[i]
        return results
